src/types/aws/ap/registration.ts

```diff
@@ -235,12 +235,7 @@ export function isDoneFSAInquiry(
 }
 
 export function isDoneDocs(data: SavedRegistration): data is DoneDocs {
-  const reg = data.Registration as Partial<TDocumentation>;
```

Comment:
revert to - `docs` is complete if user uploaded all necessary files,info to generate signing URL